tests/integration/clientv3/lease/lease_test.go

```diff
@@ -117,6 +117,21 @@ func TestLeaseKeepAliveOnce(t *testing.T) {
 	if err != rpctypes.ErrLeaseNotFound {
 		t.Errorf("expected %v, got %v", rpctypes.ErrLeaseNotFound, err)
 	}
+
+	clus.Members[1].Stop(t)
```

Comment:
nit: I think it's better to move this to a separate test.

Reply:
You mean another TestLeaseKeepAliveOnce func? Not sure if we shhould do that in integeration testing

Reply:
Copy the `TestLeaseKeepAliveOnce` test and rename it to `TestLeaseKeepAliveOnceNoLeader`